_posts/course-materials/earth-analytics/week-12/in-class/2017-04-19-social-media-03-r.md

```diff
@@ -118,7 +114,9 @@ those.
 
 # remove http elements manually
 climate_tweets$stripped_text <- gsub("http.*","",  climate_tweets$text)
+## Error in gsub("http.*", "", climate_tweets$text): object 'climate_tweets' not found
```

Comment:
Is this error and the ones below going to be fixed with the other changes you mentioned?